src/Terrain/Environment.java

```diff
@@ -54,7 +54,7 @@ public int getDefLevel() {
 		case FACTORY:
 			return 4;
 		case HQ:
-			return 5;
+			return 4;
```

Comment:
Unrelated change, yes, but I didn't wanna bother with making an issue or a whole new branch for literally one character.